resources/pvc.py

```diff
@@ -102,7 +103,7 @@ def get_all_pvcs():
     return out
 
 
-def create_multiple_pvc(number_of_pvc=1, pvc_data=CSI_PVC_DICT.copy()):
+def create_multiple_pvc(number_of_pvc=1, pvc_data=copy.deepcopy(CSI_PVC_DICT)):
```

Comment:
Don't use mutable function defaults please!
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

An alternative is to use a default of `None`, then early in the function body:
`pvc_data = pvc_data or copy.deepcopy(CSI_PVC_DICT)`

Reply:
Ah, good point. I just run the change via sed and haven't noticed this problem.

Reply:
I'm going to add https://github.com/ebeweber/flake8-mutable to our flake8 run.

Reply:
Excellent! I didn't know that existed.

Reply:
Addressed in new commits.

* The [new flake8 run catched the problem](https://travis-ci.org/red-hat-storage/ocs-ci/jobs/548303698)
* And then the fix in commit 8ed5847d8024fd828821000cc8fad6a9dcbfeac9 [makes flake8 pass again](https://travis-ci.org/red-hat-storage/ocs-ci/jobs/548305102)